src/pages/blog/components/no-results/NoResults.tsx

```diff
@@ -0,0 +1,17 @@
+import Typography from "@mui/material/Typography";
+
+import { BoxStyled } from "@/pages/blog/components/no-results/NoResults.styled";
+
+export default function NoResults() {
+  return (
+    <BoxStyled>
+      {/* TODO: replace Typography with AppTypography and fix ui */}
```

Comment:
Job job! And when do we plan to replace it?

Reply:
Replaced with AppTypography